WordPress/gutenberg
# Video: Add a deprecation for the caption element

## What?
Adds a deprecation for the video block caption element class name. I should have done this in https://github.com/WordPress/gutenberg/pull/41140.

## Why?
When we change the save function of a block we need to add a deprecation so that the new code knows how to interpret the old markup.

## How?
This just adds a copy of the save function and attributes of the block in the state they were in before https://github.com/WordPress/gutenberg/pull/41140.

## Testing Instructions
- Using WordPress 6.0, add a video block to a post
- Activate Gutenberg
- Reload the post editor
- Confirm that you see a block validation error:
<img width="785" alt="Screenshot 2022-10-20 at 16 43 30" src="https://user-images.githubusercontent.com/275961/197001466-e3b446a1-8896-4929-8698-52e596a4fcca.png">

- Checkout this branch
- Confirm that the error is gone: 
<img width="796" alt="Screenshot 2022-10-20 at 17 01 00" src="https://user-images.githubusercontent.com/275961/197001496-cbe077f5-755e-40b6-b486-724f387aa919.png">
